packages/components/package.json

```diff
@@ -31,6 +31,7 @@
     "build:types": "tsc --project . --declaration --emitDeclarationOnly",
     "build:watch": "yarn run build:js --watch",
     "build:storybook": "build-storybook -o storybook-static",
+    "chromatic": "chromatic --project-token=CHROMATIC_PROJECT_TOKEN --skip 'dependabot/**'",
```

Comment:
Is this hooked up to the GH action automagically? or is it for local use?

Reply:
Automagically. Pretty sure Chromatic's CI integration expects a `chromatic` script to be present.